core/src/main/java/org/elasticsearch/action/admin/indices/analyze/AnalyzeResponse.java

```diff
@@ -108,6 +117,12 @@ public void readFrom(StreamInput in) throws IOException {
             startOffset = in.readInt();
             endOffset = in.readInt();
             position = in.readVInt();
+            Integer len = in.readOptionalVInt();
```

Comment:
I guess we could possibly read from a 5.x node less than I hope 5.2.  I can add a check, should I do 5.2 or 5.3?